### PR TITLE
Add export_bundle and download_bundle methods

### PR DESCRIPTION
**Pull request by Halle**
## Description

This adds two not-yet-supported CrowdIn API bundle operation methods to `crowdin-api-client-ruby`, which supply the ability to export a defined bundle and then to download the exported bundle.

## Motivation

**Target File Bundles** are how **CrowdIn** allows the export of translation sets in different string formats. In order to be able to use one source file or source "vault" to be able to export either iOS (.string) or Android (.xml) string formats, we first define the output bundle types on **CrowdIn** and then download them manually. In order to be able to make this part of Ruby automation, we need to add support for these new API methods to `crowdin-api-client-ruby`.

## Unit Testing

I added the test specs in their preferred format, on the theory that we may want to submit a PR to the main project at some point.

## Manual Testing

I have used these methods and they work.

## Ready to Review

All of the below must pass 100% before marking the PR as Ready to Review.  Any exceptions should be explained here or in the PR comments.

- ✅  All pre-submit unit tests pass
- ✅  All linter and static analysis warnings addressed
- ✅  All temporary testing code removed

## Before Merging

1. Merge the base branch (usually develop) into this branch first
2. Rerun all formatters, linters, and pre-sbmit unit tests and ensure all pass

### CODE REVIEW

File: lib/crowdin-api/api_resources/bundles.rb [hunk 1]
```diff
@@ -36,6 +36,38 @@ def add_bundle(query = {}, project_id = config.project_id)
...
+        
+        request = Web::Request.new(
+          connection,
+          :post,
+          "#{config.target_api_url}/projects/#{project_id}/bundles/#{bundle_id}/exports",
+          { params: query }
+        )
+        Web::SendRequest.new(request).perform
+      end
+
```
Comment (Halle) [inline lib/crowdin-api/api_resources/bundles.rb:54]: This allows a pre-formatted Target File Bundle to be exported within CrowdIn's file system in preparation for its later download.

File: lib/crowdin-api/api_resources/bundles.rb [hunk 2]
```diff
@@ -36,6 +36,38 @@ def add_bundle(query = {}, project_id = config.project_id)
...
+        project_id || raise_project_id_is_required_error
+
+        request = Web::Request.new(
+          connection,
+          :get,
+          "#{config.target_api_url}/projects/#{project_id}/bundles/#{bundle_id}/exports/#{export_id}/download"
+        )
+        Web::SendRequest.new(request).perform
+      end
+
```
Comment (Halle) [inline lib/crowdin-api/api_resources/bundles.rb:70]: This allows the exported bundle to be downloaded.

File: spec/api_resources/bundles_spec.rb [hunk 3]
```diff
@@ -18,6 +18,22 @@
...
+    end
+    
+    describe '#download_bundle' do
+      it 'when request are valid', :default do
+        stub_request(:get, "https://api.crowdin.com/#{target_api_url}/projects/#{project_id}/bundles/#{bundle_id}/exports/#{export_id}/download")
+        download_bundle = @crowdin.download_bundle(project_id, bundle_id, export_id)
+        expect(download_bundle).to eq(200)
+      end
+    end
+
```
Comment (Halle) [inline spec/api_resources/bundles_spec.rb:36]: If we want to make a PR in the future I believe this supplies their project requirements.

File: spec/unit/client_spec.rb [hunk 4]
```diff
@@ -88,4 +88,10 @@
       expect { @crowdin.fetch_all(:add_bundle).to raise_error(Crowdin::Errors::FetchAllProcessingError) }
     end
   end
+  
+    describe 'Crowdin Client fetch_all' do
+    it 'should raise error if fetch_all is called for unsupported methods' do
+      expect { @crowdin.fetch_all(:export_bundle).to raise_error(Crowdin::Errors::FetchAllProcessingError) }
+    end
```
Comment (Halle) [inline spec/unit/client_spec.rb:95]: Also project needs.

**Review by waroo — APPROVED** [review]

**Pull request merged**